<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addsite</class>
 <widget class="QMainWindow" name="addsite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>901</width>
      <height>751</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string>添加景点</string>
    </property>
    <widget class="QTextEdit" name="name">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>景点ID</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ticketnum">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>景点名称</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="id">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>景点票价</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>景点门票量</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>130</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1752</year>
       <month>9</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>MM/dd</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>旺季由每年</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>旺季票价</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>至</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>淡季票价</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pricehigh">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="pricelow">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>特殊票价</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>添加...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>学生</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>军人</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>440</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>景点级别</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="level">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>440</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>130</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>MM/dd</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="profile">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>560</y>
       <width>561</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>380</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>维护费用</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>560</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>景点简介</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="maprice">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>380</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="timelimit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>330</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加人数限制时段</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>440</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>所在区域</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="area">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>430</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>510</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>游览所需时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>分钟</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="time">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>500</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="newid">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="newlabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设置折扣为</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>650</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(200字符以内)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveprice">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>200</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QLabel" name="idprompt">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>必须为6位数字</string>
     </property>
    </widget>
    <widget class="QPushButton" name="publish">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>690</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发布</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>700</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>700</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>保存草稿</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newprice">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="disprompt">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>240</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>折扣格式不正确</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>500</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>所有人</string>
     </property>
     <item>
      <property name="text">
       <string>所有人</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>不适合儿童</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>不适合长者</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>不适合儿童及长者</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>510</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>适合人群</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
